markdown/developers/adding_pipelines.md

```diff
@@ -8,7 +8,7 @@ subtitle: Follow this walkthrough to add a new pipeline to nf-core.
 So, you want to add a new pipeline to nf-core - brilliant!
 Before you start typing, check that you're happy with the following points:
 
-* You're familiar with nf-core and nextflow (see our [introduction docs](/usage/introduction)).
+* You're familiar with nf-core and nextflow (see our [introduction docs](../usage/introduction.md)).
```

Comment:
This change is fine.. 👍 

Going from https://nf-co.re/developers/adding_pipelines the link becomes https://nf-co.re/usage/introduction.md which correctly redirects to https://nf-co.re/usage/introduction